libmscore/mscore.h

```diff
@@ -88,7 +88,7 @@ static constexpr qreal INCH      = 25.4;
 static constexpr qreal PPI       = 72.0;           // printer points per inch
 static constexpr qreal DPI_F     = 5;
 static constexpr qreal DPI       = 72.0 * DPI_F;
-static constexpr qreal SPATIUM20 = 5.0 * (DPI / 72.0);
+static constexpr qreal SPATIUM20 = 24.8;
```

Comment:
why not leave the calculation? it's a constexpr anyway so calculated at compile time.

Reply:
Yes, I agree. It is just self documenting the way it was